### ISSUE TITLE
Consider only L1 loss

### ISSUE THREAD

**Issue by riktimmondal** [mrzhu-cool/pix2pix-pytorch#11]
I want to consider only L1 loss.Is it correct to comment out the generator loss  as 

#loss_g = loss_g_gan + loss_g_l1
and consider only, 
loss_g = loss_g_l1

Do I need to change anything in discriminator part?

Thanks in advance

**Comment by andreaschandra** [comment]
any update on this?

**Comment by mrzhu-cool** [comment]
Yes, you don't need to change any other things. Just loss_g = loss_g_l1.

**Comment by olivier-gillet** [comment]
Did you get good results with it @riktimmondal  ?
